symplyphysics/laws/hydro/bernoullis_equation.py

```diff
@@ -0,0 +1,42 @@
+from sympy import Eq, solve, dsolve, Derivative
```

Comment:
Don't use marketing names (like Bernoulli's equation) for law. Describe in the name what you get from what you put.